Documentation/policy/language.rst

```diff
@@ -414,6 +419,75 @@ but not CIDR prefix ``10.96.0.0/12``
 
         .. literalinclude:: ../../examples/policies/l3/cidr/cidr.json
 
+.. _DNS based:
+
+DNS based
+---------
+
+``toFQDNs`` simplifies specifying egress policy to IPs of remote, external,
+peers. The DNS lookup for each ``matchName`` is done periodically by
+``cilium-agent`` and the result is used to regenerate endpoint policy. This
+allows tracking changing IPs or sets of IPs that may not be known a priori.
+Despite the naming, the ``matchName`` field does not have to be a
+fully-qualified domain name. In cases where search domains are configured, the
+DNS lookups from ``cilium`` will not be qualified and will utilize the search
+list.
+
+The DNS lookups are repeated with an interval of 5 seconds, and are made for
+A(IPv4) and AAAA(IPv6) addresses. Should a lookup fail, the most recent IP data
+is used instead. An IP change will trigger a regeneration of the ``cilium``
+policy for each endpoint, and the updated IPs can be seen in the response from
+``cilium policy get``. Each update will also increment the per ``cilium-agent``
```

Comment:
This is completely unrelated to this PR, but I think it'd be helpful for users if we could have some type of CLI command which shows the IPs that are mapped to an DNS hostname, e.g. "cilium fqdn-map list" and then that shows the mapping of hostname to IPs based off of the modified policy rules in cilium. This was just an idea, but I can create a GH issue for this if you think it is worthwhile.